libraries/app/include/graphene/app/api.hpp

```diff
@@ -176,10 +176,31 @@ namespace graphene { namespace app {
                                                                         uint32_t stop = 0,
                                                                         unsigned limit = 100,
                                                                         uint32_t start = 0) const;
-
+         /**
+          * @brief Get orders with b:a pair
+          * @param a Base in pair
+          * @param b Quote in pair
+          * @param limit Maximum number of orders to retrieve
+          * @return A list of orders with b:a pair
+          */
          vector<order_history_object> get_fill_order_history( asset_id_type a, asset_id_type b, uint32_t limit )const;
+
+         /**
+          * @brief Get markets with b:a pair
+          * @param a Base in pair
+          * @param b Quote in pair
+          * @param bucket_seconds Represents the length in seconds of bucket
+          * @param start Time in seconds from where buckets begin
+          * @param end Time in seconds where buckets stop
+          * @return A list of markets with b:a pair (maximum markets to retrieve is less than 200)
```

Comment:
It returns history in one market, not markets. Descriptions of params seems off.